src/inputs/internal/SelectFieldInput.tsx

```diff
@@ -156,6 +156,11 @@ export function SelectFieldInput<O, V extends Value>(props: SelectFieldInputProp
             if (inputProps.readOnly) return;
             setIsFocused(true);
             maybeCall(onFocus);
+          },
+          onClick: () => {
+            if (inputProps.readOnly) return;
+            setIsFocused(true);
```

Comment:
**question** Do we need to `setIsFocused` to true here? I'd assume once a user clicks on the input, then `focus` will also be triggered and that'll call `setIsFocused(true)`.